docs/core/windows-prerequisites.md

```diff
@@ -13,45 +13,117 @@ ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
+
+However, if you are installing .NET Core 2.0 via the [installer script](./tools/dotnet-install-script.md), or deploying a self-contained .NET Core 2.0 application; you need to manually install the [Microsoft Visual C++ 2015 Redistributable Update 3](https://www.microsoft.com/en-us/download/details.aspx?id=52685).
+
+> [!NOTE]
+> <em>For Windows 7 and Windows Server 2008 machines only:</em><br>
+> Make sure that your Windows installation is up-to-date and includes hotfix [KB2533623](https://support.microsoft.com/help/2533623) installed through Windows Update.
+
+## Developing .NET Core Applications in Visual Studio 2017
+
+You can use any editor of your choice to develop .NET Core applications using the .NET Core SDK. However, if you want to develop .NET Core applications on Windows in an integrated development environment, you can use [Visual Studio 2017](#visual-studio-2017).
+
+There are different editions of Visual Studio 2017. You can download [Visual Studio Community 2017](https://www.visualstudio.com/downloads/) for free to get started.  
 
-There are different editions of Visual Studio 2017. You can download [Visual Studio Community 2017](https://www.visualstudio.com/downloads/) for free to get started.  To learn more about the Visual Studio installation process, see [Install Visual Studio 2017](/visualstudio/install/install-visual-studio).
+To learn more about the Visual Studio installation process, see [Install Visual Studio 2017](/visualstudio/install/install-visual-studio).
 
 To verify that you're running the latest version of Visual Studio 2017, do the following:
 
  * On the **Help** menu, choose **About Microsoft Visual Studio**.
  * In the **About Microsoft Visual Studio** dialog, the version number should be 15.0.26228.4 or higher.
 
 You can read more about the changes in Visual Studio 2017 in the [release notes](https://www.visualstudio.com/news/releasenotes/vs2017-relnotes).
+
+[.NET Core 1.x](#tab/netcore1x)
+
+> [!IMPORTANT]
+> Even though, it's possible to use Visual Studio 2015 with a preview version of the .NET Core tooling, these projects are *project.json*-based, which is now deprecated. Visual Studio 2017 uses project files based on MSBuild. For more information about the format changes, see [High-level overview of changes](./tools/cli-msbuild-architecture.md).
+
+To develop .NET Core 1.x apps in Visual Studio, install the [latest version of Visual Studio](https://www.visualstudio.com/downloads/) with the **.NET Core cross-platform development** toolset (in the **Other Toolsets** section) selected.
+
+![Screenshot of Visual Studio 2017 installation with the ".NET Core cross-platform development" workload selected](./media/windows-prerequisites/vs_workloads.jpg)
+
+     To verify that you're running the latest version of Visual Studio 2017, do the following:
```

Comment:
This won't be true in a couple months - maybe sooner - consider
To verify Visual Studio 2017 is version number 15.0.26228.4 or later
 
   * On the **Help** menu, choose **About Microsoft Visual Studio**.
   * Check the version number in the **About Microsoft Visual Studio**.